deploy/crds/build.dev_builds_crd.yaml

```diff
@@ -1,225 +1,225 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Hi @sbose78 ,

A quick question, how did you generate this crds?

I tried to run the commands:
```
operator-sdk generate k8s
operator-sdk generate crds
```

But there is no change in my side. 

Did you change to use the latest operator-sdk `v0.18.x` or how you make this change?

Reply:
same question @sbose78

Reply:
This was definitely an unintended changed :) My intention was to only modify
https://github.com/redhat-developer/build/pull/290/files#diff-2bcdec490a6ad0e944e49d3c6a9718ddR90

Reply:
Yes, I did use operator-sdk v0.18.x, thought it would be harmless ..

Reply:
Thanks Shoubhik,

So I am not sure if we can keep using the old v0.17 version operator-sdk to generate the CRDs.

And use an issue to track the kube or operator-sdk bump, like this issue: https://github.com/redhat-developer/build/issues/284

I am not sure if the v0.18 can works fine on kube 1.17, etc... so it is better that we can have some verification first and avoid the regression issue :)

Reply:
@sbose78 can u modify this PR?, I dont think we wanna be `v1` yet. We should remain with the `v1beta1`

Reply:
CRDs went GA in k8s 1.16. If we are targeting clusters higher than 1.16, we should use the GA version when possible.

Reply:
Yes, I'll make a separate PR to move to operator-sdk 0.19 and then propose moving to the GA version of CRDs.